server/sonar-server/src/main/java/org/sonar/server/branch/ws/ListAction.java

```diff
@@ -147,8 +147,8 @@ private static void addBranch(WsBranches.ListWsResponse.Builder response, Branch
     return builder;
   }
 
-  private static void setBranchStatus(WsBranches.Branch.Builder builder, BranchDto branch, @Nullable MeasureDto qualityGateMeasure, @Nullable BranchStatistics branchStatistics) {
-    WsBranches.Branch.Status.Builder statusBuilder = WsBranches.Branch.Status.newBuilder();
+  private static void setBranchStatus(ProjectBranches.Branch.Builder builder, BranchDto branch, @Nullable MeasureDto qualityGateMeasure, @Nullable BranchStatistics branchStatistics) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Split this 183 characters long line (which is greater than 180 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS00103)